install/kubernetes/helm/istio/charts/egress-gateway/templates/clusterrole.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.global.rbacEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
I know it's cut&pasted - but we should review if this is still needed. 
Gateways should not touch apiserver, just pilot

Reply:
+1 - egress gateways don't need permissions, right?